cttportfolio.css

```
@@ -49,7 +53,69 @@ a.icon_hover:hover img, a.pf_sub_links:hover h6, button:hover,.navlinks.icon_hov
 .background_color,.project_text > * {
 	background-color:rgba(255,182,193,0.7);
 	box-shadow: 0px -2px 5px rgba(2,8,9,0.6);
-	border-radius:20px;
+	border-radius:5px;
+}
+h1, h2, nav{
+    animation-fill-mode: forwards !important;
```

```suggestion
    animation-fill-mode: forwards !important;
```
what is the !important